Resources/Textures/_CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC0-1.0",
+  "copyright": "Base made by Psychpsyo, Altered into current form by KittenColony",
```

Comment:
Do you have a source for this? Or was it just provided to you privately?

Reply:
This also goes for all of the other files credited to Psychpsyo

Reply:
space asshole jacket

Reply:
Can you add that to the credits?